85% of the features just broke

Literally nothing but a few features work.
Left Side Chat works as the only thing I can see.

Even the emotes is gone. Wut?

Closed my other issue as its a duplicate from a few others that were created at same time:


Most of the current issues that are open are probably all for this issue.
Lastest version of Chrome; no extension enabled except BTTV
No antivirus or Windows network protection enabled.

DEVTOOLS -> Console -> Javascript errors

ErrorHandling.js:38 Error: apstag has already loaded - preventing duplicate load
at Object. (VM5719 apstag.js:2)
at r (VM5719 apstag.js:2)
at Object. (VM5719 apstag.js:2)
at r (VM5719 apstag.js:2)
at VM5719 apstag.js:2
at VM5719 apstag.js:2
p @ ErrorHandling.js:38
(anonymous) @ apstag.js:74
r @ bootstrap:19
(anonymous) @ Networking.js:125
r @ bootstrap:19
(anonymous) @ bootstrap:83
(anonymous) @ bootstrap:83
27Refused to load the font '' because it violates the following Content Security Policy directive: "font-src ".

component.html:1 Access to fetch at 'https://science-output.s3.amazonaws.com/spade-uri' from origin 'https://d4uvtfdr04uq6raoenvj7m86gdk16v.ext-twitch.tv' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
Store.js:243 Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self' https://mslfr4spx7byj1zchkhjvsuc5f4362.ext-twitch.tv https://extension-files.twitch.tv https://www.google-analytics.com".

--- a/src/utils/twitch.js
+++ b/src/utils/twitch.js
@@ -190,7 +190,7 @@ module.exports = {
         try {
             const node = searchReactParents(
                 getReactInstance($(REACT_ROOT)[0]),
-                n => n.stateNode.props && n.stateNode.props.history && n.stateNode.props.history.listen && n.stateNode.props.history.location
+                n => n.stateNode && n.stateNode.props && n.stateNode.props.history && n.stateNode.props.history.listen && n.stateNode.props.history.location
             );
             router = node.stateNode.props;
         } catch (_) {}

--- a/src/utils/twitch.js
+++ b/src/utils/twitch.js
@@ -188,11 +188,11 @@ module.exports = {
     getRouter() {
         let router;
         try {
-            const node = searchReactChildren(
+            const node = searchReactParents(
                 getReactInstance($(REACT_ROOT)[0]),
-                n => n.stateNode && n.stateNode.context && n.stateNode.context.router
+                n => n.stateNode.props && n.stateNode.props.history && n.stateNode.props.history.listen && n.stateNode.props.history.location
             );
-            router = node.stateNode.context.router;
+            router = node.stateNode.props;
         } catch (_) {}
 
         return router;

